pointcloud/basic.go

```diff
@@ -51,3 +51,16 @@ func (cloud *basicPointCloud) Set(p r3.Vector, d Data) error {
 func (cloud *basicPointCloud) Iterate(numBatches, myBatch int, fn func(p r3.Vector, d Data) bool) {
 	cloud.points.Iterate(numBatches, myBatch, fn)
 }
+
+// TODO: add function description.
+// VectorConvert ...
+func VectorConvert(vectors []r3.Vector) (basicPointCloud, error) {
```

Comment:
Linter is complaning:
```
unexported-return: exported func VectorConvert returns unexported type pointcloud.basicPointCloud, which can be annoying to use
```
do ya'll suggest we switch over to returning a differnt type?

Reply:
`vectorConvert` should just be an unexported function

Reply:
gotcha